docs/core-concepts/negotiation.md

```diff
@@ -0,0 +1,30 @@
+---
+id: negotiation
+title: Negotiation vs. Execution
+sidebar_label: Negotiation vs. Execution
+---
+
+⚠️ Disclaimer: This part of the documentation is still under construction!
+
+Note: Explain execution and negotiation separately on this page.
+
+Prior to the [execution of a swap](#atomic-swaps-using-htlcs) there is a negotiation phase where the two parties involved in the trade agree on what to trade (ledgers, assets) and the amounts.
```

Comment:
This depends on the application you are building on top of it :)

I would suggest do explain this kind of content in a section "Building on top of COMIT" or something like that.

Reply:
Or can you think of use cases where we use atomic swaps outside an exchange of goods; e.g. swap with oneself? 

I think there is always some kind of negotiation (or pre-execution) before an execution can start. Because one always needs a second party and has to agree with that party what to exchange. So I'd say this is legit. I can work on the wording though :)

Reply:
My point is more that the actual requirement is knowledge of all the parameters. How that happens is a detail that can be changed.

The current paragraph doesn't reflect that. Again, I think this is an artifact of trying to describe general constructs and our implementation at the same time.
Maybe we can prefix that paragraph with something like:

For COMIT to execute an atomic swap, it has to know about all the parameters. We therefore define a dedicated _negotiation phase_ that happens before the execution and is responsible for ...